docs/release-notes.md

```diff
@@ -63,6 +63,7 @@ key](https://getfedora.org/security/).
   3.5.0-experimental
 - Add RHEL for Edge spec 1.2.0-experimental, targeting Ignition spec
   3.5.0-experimental
+- Fail on invalid mointpoint configuration
```

Comment:
Let's say specifically what we're forbidding, and which specs we're forbidding it in.

Reply:
WDYT of "- Forbid relabeling mountpoints which already exist _(fcos 1.6-exp)_"?

Reply:
Wrong PR.  :grin:  This one isn't related to relabeling.

Reply:
My bad i was looking at #436 beforehand :sweat_smile:

Reply:
Changed to "Forbid mountpoints that don't have the prefix /var or /etc if with_mount_unit is true _(fcos 1.6-exp)_"

Reply:
How about this?

```
- Require `storage.filesystems.path` to start with `/etc` or `/var` if
  `with_mount_unit` is true _(fcos 1.6.0-exp, openshift 4.14.0-exp)_
```

Note hard line-wrap at 80 columns.